Scripts/createbinaries.bat

```diff
@@ -35,29 +35,27 @@ set msbuildexe=%msbuildpath% /nologo /property:Configuration=Release
 %msbuildexe% ..\EsentInterop\EsentInterop.csproj
 if errorlevel 1 goto :eof
 
-%msbuildexe% ..\EsentInterop\EsentInteropMetro.csproj
+%msbuildexe% ..\isam\Isam.csproj
 if errorlevel 1 goto :eof
 
 %msbuildexe% ..\EsentCollections\EsentCollections.csproj
 if errorlevel 1 goto :eof
 
-%msbuildexe% ..\isam\Isam.csproj
-if errorlevel 1 goto :eof
-
 @echo =-=-=-=-=-=-=-=
 @echo Copying output files to staging area...
 set dest=%~dp0tosign-%version%
 
 for %%i in ( esent.collections.dll esent.collections.pdb esent.collections.xml ) do (
-  xcopy /d ..\EsentCollections\bin\release\%%i %dest%\
+  robocopy /S ..\EsentCollections\bin\release\ %dest%\ %%i
 )

```

Comment:
Since you're converting to `robocopy`, you can get rid of the loop and use:
`robocopy /s ..\EsentCollections\bin\release\ %dest%\ esent.collections.dll esent.collections.pdb esent.collections.xml `